src/server/api/controllers/favorites.controller.js

```diff
@@ -2,53 +2,45 @@ const knex = require('../../config/db');
 const moment = require('moment-timezone');
 const HttpError = require('../lib/utils/http-error');
 
-const addToFavorites = async(body) => {
-    if (
-        body.user_id < 1 ||
-        isNaN(body.user_id) ||
-        body.product_id < 1 ||
-        isNaN(body.product_id)
-    ) {
-        throw new HttpError(
-            'Bad request. ID must be an integer and larger than 0',
-            400,
-        );
-    }
-    try {
-        await knex('favorites').insert({
-            user_id: body.user_id,
-            product_id: body.product_id,
-            created_at: moment(Date.now()).format(),
-        });
+const addToFavorites = async (body) => {
+  if (
+    body.user_id < 1 ||
+    isNaN(body.user_id) ||
+    body.product_id < 1 ||
+    isNaN(body.product_id)
+  ) {
+    throw new HttpError(
+      'Bad request. ID must be an integer and larger than 0',
+      400,
+    );
+  }
+  try {
+    await knex('favorites').insert({
+      user_id: body.user_id,
+      product_id: body.product_id,
+      created_at: moment(Date.now()).format(),
+    });
 
-        return {
-            successful: true,
-            message: 'Product was added to favorites.',
-        };
-    } catch (error) {
```

Comment:
you need to add this here so it would give us different error 
   return families;
  } catch (error) {
      if (error instanceof HttpError) {
        throw error;
      }
      throw new HttpError('Something went wrong', 500);
  }
};